packages/enzyme-adapter-react-13/src/ReactThirteenAdapter.js

```diff
@@ -242,6 +242,17 @@ class ReactThirteenAdapter extends EnzymeAdapter {
     }
   }
 
+  wrap(element) {
+    class Wrapper extends React.Component {
+      render() {
+        // eslint-disable-next-line react/prop-types
+        const { children } = this.props;
+        return children;
+      }
+    }
+    return React.createElement(Wrapper, null, element);
+  }
```

Comment:
@ljharb Could you confirm this is what you had in mind for the `wrap` function on the adapters?